ocrd_segment/repair.py

```diff
@@ -128,9 +128,13 @@ def process(self):
                         mark_for_deletion.append(region1.id)
                     elif poly1.overlaps(poly2):
                         inter_poly = poly1.intersection(poly2)
+                        union_poly = poly1.union(poly2)
                         LOG.debug('Page "%s" region "%s" overlaps "%s" by %f/%f',
                                   page_id, region1.id, region2.id, inter_poly.area/poly1.area, inter_poly.area/poly2.area)
-                        if inter_poly.area / poly2.area > self.parameter['plausibilize_merge_min_overlap']:
+                        if union_poly.convex_hull.area >= poly1.area + poly2.area:
```

Comment:
Uih, do you have an example for this case?

Reply:
Not in an image (I'd have to dig for that), I used an artificial example with constructed polygons